async_fuse/src/byte_slice.rs

```diff
@@ -91,7 +102,8 @@ impl<'a> ByteSlice<'a> {
             );
         }
 
-        if (self.data.as_ptr() as usize) % mem::align_of::<T>() != 0 {
+        let address = unsafe { mem::transmute::<*const u8, usize>(self.data.as_ptr()) };
+        if address.overflow_rem(mem::align_of::<T>()) != 0 {
```

Comment:
The same with line 69.
Consider using `wrapping_rem`.

Reply:
same above